components/curriculum/module-list.css

```diff
@@ -245,7 +245,7 @@
             align-items: center;
             justify-content: space-between;
 
-            height: 11rem;
+            height: 13rem;
```

Comment:
Isn't there be a way in flexbox to have all items take the size of the highest element? 🤔

Reply:
Oh, because these are nested, it would only work with subgrid maybe.